attribute-array/README.md

```diff
@@ -151,30 +151,24 @@ As you may have multiple versions of the runtime installed, make sure that .NET
 
 ## Getting the code
 
-There are a few ways you can start working with the code in this demo.
-
 ### **Clone the Repository to Your Local Computer:**
 
 **Using the Terminal:**
 
 - Open the terminal on your computer.
 - Navigate to the directory where you want to clone the repository.
-- Type `git clone https://github.com/Azure-Samples/cosmos-db-design-patterns.git` and press enter.
+- Type `git clone git clone https://github.com/Azure-Samples/cosmos-db-design-patterns.git` and press enter.
```

Comment:
Why is git clone here twice?

Reply:
removed git clone and just left the URL.  got it.